crypto/x509/x509_vfy.c

```diff
@@ -3042,8 +3042,6 @@ static int build_chain(X509_STORE_CTX *ctx)
      * this to change. ]
      */
     if (DANETLS_ENABLED(dane) && dane->certs != NULL) {
```

Comment:
There was a comment to why this alloc was here.. The comment needs to be updated also?

Reply:
Good point, yes.

Reply:
So the comment implies this was there where some other code changes that doesn't exist yet :)